addons/website_blog/views/website_blog_posts_loop.xml

```diff
@@ -38,11 +38,22 @@ according to the enabled options.
         <div t-attf-class="row #{posts and not opt_blog_readable and 'mx-n2'}">
             <!-- Filters -->
             <div t-if="tag or date_begin or search" class="col-12 mb-3">
+                <div t-if="posts" class="h4 mb-3">
```

Comment:
Note: the fix has been put specifically before the replaced code in order to take a possible customization into account. For example:
```.xml
<xpath expr="//div[@t-if='posts']" position="attributes">
     <attribute name="class">new_class</attribute>
</xpath>
```